src/settings.rs

```diff
@@ -28,7 +29,7 @@ impl SettingsFile {
         {
             let mut b = self.cache.borrow_mut();
             if b.is_none() {
-                *b = Some(if file::is_file(&self.path) {
+                *b = Some(if Path::is_file(&self.path) {
```

Comment:
```suggestion
                *b = Some(if self.path.is_file() {
```

Note that `PathBuf` automatically derefs to `Path` in the same way that `String` automatically derefs to `str` and `Vec<T>` automatically derefs to `[T]`. This means you can call `Path` methods directly on `PathBuf`, like so.